apps/fluent-tester/src/TestComponents/CheckboxExperimental/CheckboxTest.tsx

```diff
@@ -193,10 +193,13 @@ const checkboxSections: TestSection[] = [
     name: 'Other Implementations',
     component: OtherCheckbox,
   },
-  {
-    name: 'Token Customized Checkboxes',
-    component: TokenCheckbox,
-  },
+  Platform.select({
+    android: null,
+    default: {
+      name: 'Token Customized Checkboxes',
+      component: TokenCheckbox,
```

Comment:
Is there a reason why Android is leaving out the customized checkbox?

Reply:
@rurikoaraki , On Android platform, the design feedback was to not demonstrate the live customization (with changing states using TextFields ) as that gave surplus flexibility to developers to easily deviate from Fluent guidelines.  
Instead, the suggestion was to show one or two customized controls as indicative support for over and above Fluent design language. 
Would you have a strong opinion on removing those, incase I can revisit the design. 

FYI, 
We are in discussions with design to revamp the tester experience for Android (Given the size resolution is less than width screen endpoints) that can be also reused on iOS.  
Once that is finalized, we can revisit the 'customized' section for Android controls and show what design suggests.

Reply:
Sounds good to me since we would still be showing few examples to indicate customized controls can exist, but just going away from highlighting the support as a play demo. I would defer the final call based on if Ruriko has a opinion against this.

Reply:
Sure that sounds fine. It's probably worth talking to @ejlayne about since customization is something that we offer, so we should probably align across platforms how we treat customization.